text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/MiniMessageParser.java

```diff
@@ -247,4 +250,10 @@ private Component handleModifying(final Modifying modTransformation, final Compo
   private String sanitizePlaceholderName(final String name) {
     return name.toLowerCase(Locale.ROOT);
   }
+
+  private void tryToNag(final String input, final boolean strict) {
```

Comment:
Maybe just call this in `TokenParser`? I believe that would capture all code paths at once.

Reply:
Yeah, perhaps doing it as late as possible is the better solution?